src/elements/content-sidebar/activity-feed/annotation-thread/__mocks__/replies.ts

```diff
@@ -0,0 +1,81 @@
+export const user = {
+    id: '1',
+    login: 'johndoe',
+    name: 'John Doe',
+    type: 'user' as const,
+};
+
+export const replies = [
```

Comment:
I thought I'd seen another fixture file for replies. Let's be sure we're not duplicating our fixtures unless we have reason.